models/articles-model.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection");
+
+exports.selectArticles = (article_id) => {
+  if (!Number(article_id)) {
```

Comment:
Psql has specific codes for checking if a query value is the right data type. There is no need for a custom reject.